### PR TITLE
log(reputation): fix reputation logging in Live state

### PR DESCRIPTION
**Pull request by lrubiorod**

### CODE REVIEW

File: node/src/actors/chain_manager/mod.rs [hunk 1]
```diff
@@ -445,7 +445,7 @@ impl ChainManager {
 
                 // Print reputation logs on debug level on synced state,
```
Comment (mariocao) [inline node/src/actors/chain_manager/mod.rs:446]:
```suggestion
                // Print reputation logs on debug level on Live state,
```

File: node/src/actors/chain_manager/mod.rs [hunk 2]
```diff
@@ -1058,27 +1058,28 @@ fn update_reputation(
     {
         log::error!("Error updating reputation in consolidation: {}", e);
     }
+    if log_level != log::Level::Trace {
```
Comment (tmpolaczyk) [inline node/src/actors/chain_manager/mod.rs:1061]: We can also skip this if the current log level is INFO or ERROR, so try doing:
```suggestion
    if log_level >= log::max_level() {
```

Comment (tmpolaczyk) [reply node/src/actors/chain_manager/mod.rs:1061]: Even better:
```suggestion
    if log_enabled!(log_level) {
```

File: node/src/actors/chain_manager/mod.rs [hunk 3]
```diff
@@ -1058,27 +1058,28 @@ fn update_reputation(
     {
         log::error!("Error updating reputation in consolidation: {}", e);
     }
+    if log::log_enabled!(log_level) {
```
Comment (aesedepece) [inline node/src/actors/chain_manager/mod.rs:1061]: Maybe this:
```suggestion

    if log::log_enabled!(log_level) {
```

File: node/src/actors/chain_manager/mod.rs [hunk 4]
```diff
@@ -1069,26 +1069,28 @@ fn update_reputation(
...
-            ' '
-        };
-        log::log!(log_level, "    [{}] {}: {}", active, pkh, rep.0);
+        log::log!(log_level, "Total Reputation: {{");
+        for (pkh, rep) in rep_eng
+            .trs()
+            .identities()
+            .sorted_by_key(|&(_, &r)| std::cmp::Reverse(r))
+        {
+            let active = if rep_eng.ars().contains(pkh) {
```
Comment (aesedepece) [inline node/src/actors/chain_manager/mod.rs:1085]:
```suggestion
            let active = if rep_eng.is_ars_member(pkh) {
```

**Pull request merged**